server/pulp/server/webservices/http.py

```diff
@@ -210,7 +154,7 @@ def ssl_client_cert():
     @rtype: str or None
     @return: pem encoded cert
     """
-    return _get_wsgi_environ().get('SSL_CLIENT_CERT', None)
+    return _thread_local.wsgi_environ.get('SSL_CLIENT_CERT', None)
```

Comment:
This is resolved with the inclusion of SaveEnvironWSGIHandler in the last force push.